src/test/kotlin/com/pubnub/contract/uuidmetadata/state/UUIDMetadataState.kt

```diff
@@ -0,0 +1,10 @@
+package com.pubnub.contract.uuidmetadata.state
+
+import com.pubnub.api.models.consumer.objects.uuid.PNUUIDMetadata
+
+class UUIDMetadataState {
+    var uuidMetadatas: Collection<PNUUIDMetadata>? = null
+    var uuidMetadata: PNUUIDMetadata? = null
+    var uuid: String? = null
+    var name: String? = null
```

Comment:
is name needed?

Reply:
I'll check it. It might be a leftover from some older version